packages/react-monaco-editor/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui/react-monaco-editor",
-  "version": "0.13.18",
+  "version": "1.0.0-beta",
```

Comment:
This one I'm also less enthusiastic about bumping to 1.0, but it's probably unavoidable